app/routes.py

```diff
@@ -1 +1,275 @@
-from flask import Blueprint
\ No newline at end of file
+import json
+from attr import validate
+from flask import Blueprint, jsonify, request, abort, make_response
+import sqlalchemy
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from sqlalchemy import desc
+from datetime import datetime
+import requests
+import os
+from dotenv import load_dotenv
+
+# load_dotenv()
+
+tasks_bp = Blueprint('tasks_bp', __name__, url_prefix='/tasks')
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+@tasks_bp.route('', methods=['POST'])
+def create_one_task():
+    request_body = request.get_json()
+    
+    if "description" not in request_body.keys() or "title" not in request_body.keys():
+        return make_response({"details": "Invalid data"}, 400) 
+
+    
+    else:
+        new_task = Task(
+            title=request_body["title"], 
+            description=request_body["description"],
+            completed_at=request_body["completed_at"] if "completed_at" in request_body else None)
+        
+        db.session.add(new_task)
+        db.session.commit()
+        return jsonify({"task": {
+                    "id": new_task.task_id,
+                    "title": new_task.title,
+                    "description": new_task.description,
+                    "is_complete": bool(new_task.completed_at)
+                }}), 201
```

Comment:
The `get_dict` method will do this work for you:
```suggestion
        return jsonify({"task": new_task.get_dict()}), 201
```